torch/_export/__init__.py

```diff
@@ -302,6 +302,7 @@ def optimized(*args, **kwargs):
         in_spec = pytree.treespec_loads(call_spec[0])
         out_spec = pytree.treespec_loads(call_spec[1])
         flat_inputs = pytree.tree_flatten((args, reorder_kwargs(kwargs, in_spec)))[0]
+        flat_inputs = [x for x in flat_inputs if isinstance(x, torch.Tensor)]
```

Comment:
Hmm, right now it results in a type error if you pass non-tensors to the runner because the cpp interface is for a List[Tensor]. Does this mean that we should convert it to being a union of inputs?